nemo/collections/tts/data/tts_dataset.py

```diff
@@ -195,7 +194,6 @@ def __init__(
                 raise ValueError(f"tokens must be specified if text_tokenizer is not BaseTokenizer")
 
             self.text_tokenizer_pad_id = text_tokenizer_pad_id
-            self.tokens = tokens
```

Comment:
this was not used anywhere, so that's why you remove it. Is my understanding correct?

Reply:
Yep--same as above.